addons/bus/models/ir_websocket.py

```diff
@@ -63,7 +63,7 @@ def _build_bus_channel_list(self, channels):
         """
         req = request or wsrequest
         channels.append('broadcast')
-        channels.extend(self.env.user.groups_id)
+        channels.extend(self.env.user.group_ids)
```

Comment:
Counter-part comment about my enterprise remark: we might want to use `all_group_ids` here (we want to target all groups the user can access, implied groups included).

It might not be the case even before the PR, but that was an oversight due to the complexity to figure out what's happening (props to this PR for resolving it).

Reply:
Thank you for the feedback, but no, we are not going to change it, and no, it is not a bad design.
We want to _specifically_ target security group, because what we're sending is something that only people with this specific access should receive.
It's not used for mailing list, it's used to bus notify users of specific groups, without having to pollute the bus table with one notification for each user of the group (on top of having to actually fetch all users, which doesn't scale), when one entry for the whole group is sufficient and works more efficiently.